docker-compose.yaml

```diff
@@ -55,18 +55,28 @@ services:
   # The CDN system's main entry point; where users contact first.
   marshal-0:
     environment:
-      RUST_LOG: info
+      - RUST_LOG
     image: ghcr.io/espressosystems/espresso-sequencer/cdn-marshal:main
     command:
       - -d
       - redis://:changemeplease!!@keydb:6379
       - --bind-port 
       - $ESPRESSO_CDN_SERVER_PORT
+      - --metrics-enabled
+    healthcheck:
```

Comment:
Better to put this healthcheck in the Dockerfile itself so we can still use it even with different orchestration

Reply:
Are we also moving it out of here? My worry is that if we want to change the port there, we would have to rebuild the Dockerfile, so in the compose would not respect the environment variable. 

We could also change the compose to build the images instead of pulling them and then pass in the environment variable there.

Reply:
You can reference environment variables inside the Dockerfile in the healthcheck command, and if they are overridden at runtime the command will respect that